### PR TITLE
Add dablooms spec and dablooms-toolfile spec so dablooms library can be linked by StaticAnalyzers library

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
@cmsbuild please test

**Comment by smuzaffar** [comment]
please test

File: py2-dablooms.spec [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+%prep
+%setup -n dablooms-%realversion
+
+%build
+make; cd pydablooms; python setup.py build
+
+
+%install
+cd pydablooms; python setup.py install --prefix=%i
```
Comment (smuzaffar) [inline py2-dablooms.spec]: looks like we are not installing libdablooms.so. I only see python module being copied to the install area.
```
-bash-4.1$ pwd
slc6_amd64_gcc700/external/py2-dablooms/0.9.1-cms2/lib
-bash-4.1$ find . 
./python2.7
./python2.7/site-packages
./python2.7/site-packages/pydablooms.so
./python2.7/site-packages/pydablooms-0.9.1-py2.7.egg-info
```

**Comment by gartung** [comment]
I moved the dablooms library build to its own spec and toolfile spec.

**Comment by gartung** [comment]
I was not sure that I could have a python package that also provided a library to link to.

**Comment by smuzaffar** [comment]
you do not need separate spec for that just run `make install prefix=%i` in the `%install` section

File: python_tools.spec [hunk 2]
```diff
@@ -140,7 +140,7 @@ Requires: py2-matplotlib
 Requires: py2-numpy-toolfile
 Requires: py2-sqlalchemy
 Requires: py2-pygithub
-#Requires: py2-dablooms
+Requires: py2-dablooms
```
Comment (smuzaffar) [inline python_tools.spec]: make this `py2-dablooms-toolfile` and remove it from cmssw-tool-conf

**Comment by gartung** [comment]
please test

**Comment by smuzaffar** [comment]
@gartung , I have started the test with cms-sw/cmssw#24949 to make sure that pydablooms is available in cmssw env.

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**